### PR TITLE
[Merged by Bors] - Update vendored Crevice to 0.8.0 + PR for arrays

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- Update vendor crevice to have the latest update from crevice 0.8.0
- Using https://github.com/ElectronicRU/crevice/tree/arrays which has the changes to make arrays work

## Solution

- Also updated glam and hexasphere to only have one version of glam
- From the original PR, using crevice to write GLSL code containing arrays would probably not work but it's not something used by Bevy

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -108,7 +108,7 @@ ron = "0.6.2"
 serde = { version = "1", features = ["derive"] }
 # Needed to poll Task examples
 futures-lite = "1.11.3"
-crevice = {path = "crates/crevice"}
+crevice = { path = "crates/crevice", version = "0.8.0", features = ["glam"] }
```
Comment (bjorn3) [inline Cargo.toml:111]: Specifying a version here will make cargo use the non-vendored version when publishing.

Comment (mockersf) [reply Cargo.toml:111]: There are 3 dependencies on crevice in Bevy crates, it was the only one not specifying a version so I added it for consistency.

Before Bevy release, Cart will also have to rename it to "bevy_crevice" or something

Comment (superdump) [reply Cargo.toml:111]: That depends whether we even need to vendor it anymore. I think we should at least try it from crates.io after the recent rewrite and see if it fits our needs or not.

Comment (mockersf) [reply Cargo.toml:111]: I tried, it's missing the array PR. once that's merged and release we wouldn't need to vendor it

Comment (superdump) [reply Cargo.toml:111]: Perfect! :)

**Pull request closed without merging**